db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+puts "Seeding..."
+Category.create(name: 'Excersize')
+Category.create(name: 'Education')
+Category.create(name: 'Recipe')
```

Comment:
Kindly use for loop to create categories.

Reply:
used loop and followed dry principle of rails .